### PR TITLE
Feature/toast placement

### PR DESCRIPTION
**Pull request by pthoang**
## Description
Fixes [DG-370](https://payexjira.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=DG&modal=detail&selectedIssue=DG-370)

### CODE REVIEW

**Comment by pthoang** [comment]
I removed the dynamic sticky functionality for now, as it introduced trouble with toast on mobile. The issues with toast on IE is still present

File: src/scripts/main/toast/index.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ class Toast {
             type: "",
             icon: "",
             dismissable: true,
-            displayLength: 4000,
+            displayLength: 400000,
```
Comment (VikingTristan) [inline src/scripts/main/toast/index.js]: Hmm 🤔 I guess this remains from testing, or? :)

Comment (pthoang) [reply src/scripts/main/toast/index.js]: Yup, haha, forgot about that

File: src/scripts/main/toast/index.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { extendObj } from "../utils";
+import { callbackify } from "util";
```
Comment (VikingTristan) [inline src/scripts/main/toast/index.js]: I'm not sure about this import, but I think it can be removed and then we can merge this PR?

Comment (pthoang) [reply src/scripts/main/toast/index.js]: Agree. Not sure how it ended up there, will remove ASAP

**Pull request merged**